src/main/java/com/github/technus/tectech/compatibility/dreamcraft/DreamCraftRecipeLoader.java

```diff
@@ -5019,46 +5019,111 @@ private void addEOHRecipes() {
                 ? FluidRegistry.getFluid("molten.mutatedlivingsolder")
                 : FluidRegistry.getFluid("molten.solderingalloy");
 
+        ItemStack largeShirabonPlate = getModItem("TGregworks", "tGregToolPartLargePlate", 1, 1735);
+        // NBTTagCompound tag = largeShirabonPlate.getTagCompound();
+        // tag.setString("material", "Shirabon");
+        // largeShirabonPlate.writeToNBT(tag);
+
+        final FluidStack[] specialFluid = new FluidStack[] { FluidUtils.getFluidStack("molten.shirabon", 1_440),
+                Materials.WhiteDwarfMatter.getMolten(1_440), Materials.WhiteDwarfMatter.getMolten(1_440 * 4),
+                Materials.WhiteDwarfMatter.getMolten(1_440 * 16), Materials.BlackDwarfMatter.getMolten(1_440),
+                Materials.BlackDwarfMatter.getMolten(1_440 * 4), Materials.BlackDwarfMatter.getMolten(1_440 * 16),
+                Materials.BlackDwarfMatter.getMolten(1_440 * 64),
+                Materials.MagnetohydrodynamicallyConstrainedStarMatter.getMolten(1_440) };
+
+        final ItemStack[] plateList = new ItemStack[] {
+                // Dense Shirabon plate.
+                getModItem("bartworks", "gt.bwMetaGeneratedplateDense", 1, 10112),
```

Comment:
GT_OreDictUnificator.get("oredict", amount).
As an example in here it would be: GT_OreDictUnificator.get("plateDenseShirabon", 1) instead of magic number. Note: i am not sure if that is the correct oredict. make sure to check it!